Kitodo/src/main/java/org/kitodo/production/model/bibliography/course/Block.java

```diff
@@ -240,8 +296,57 @@ public LocalDate getLastAppearance() {
     }
 
     /**
-     * Returns whether the block is in an empty state or
-     * not.
+     * Returns the metadata assigned to this block.
+     *
+     * @return the metadata
+     */
+    public Collection<CountableMetadata> getMetadata() {
+        return metadata;
+    }
+
+    /**
+     * Returns all metadata counters from this block for the given metadataType
+     * that starts on the given day.
+     *
+     * @param metadataType
+     *            metadataType to compare
+     * @param issue
+     *            creation point to compare
+     * @param create
+     *            if the metadata was created (else deleted)
+     * @return true, if there is such a counter
+     */
+    public CountableMetadata getMetadata(String metadataType, Pair<LocalDate, Issue> issue, boolean create) {
+        for (CountableMetadata metaDatum : metadata) {
+            if (metaDatum.matches(metadataType, issue, create)) {
+                return metaDatum;
+            }
+        }
+        return null;
```

Comment:
Is there no other way to return here something more meaningful than null? Some exact value which says  that there was no countable metadata? In general this method is confusing - its JavaDoc says that it returns true.